packages/storybook/src/stories/grid-rendering.stories.tsx

```diff
@@ -13,17 +13,16 @@ export default {
   },
 };
 export const RenderInputInCell = () => {
-  const size: ElementSize = { width: 800, height: 600 };
-
-  const handleInputKeyDown = () => {};
+  // eslint-disable-next-line no-console
+  const handleInputKeyDown = (event) => console.log(event.target.value);
```

Comment:
you can use the action addon on this 
ie `action(InputChange)(event.target.value)`